tests/NHapi.NUnit/Parser/LegacyPipeParserV21Tests.cs

```diff
@@ -54,6 +54,28 @@ public void TestObx5DataTypeIsSetFromObx2_AndAllDataTypesAreConstructable(Type e
             Assert.IsAssignableFrom(expectedObservationValueType, actualObservationValueInstance);
         }
 
+        [Test]
+        public void TestObx5DataTypeIsSetFromObx2_WhenObx2IsEmptyAndDefaultIsSet_DefaultTypeIsUsed()
+        {
+            var expectedObservationValueType = typeof(ST);
+
+            var message = "MSH|^~\\&|XPress Arrival||||200610120839||ORU^R01|EBzH1711114101206|P|2.1|||AL|||ASCII\r"
+                        + "PID|1||1711114||Appt^Test||19720501||||||||||||001020006\r"
+                        + "ORC|||||F\r"
+                        + "OBR|1|||ehipack^eHippa Acknowlegment|||200610120839|||||||||00002^eProvider^Electronic|||||||||F\r"
+                        + "OBX|1||||STValue||||||F\r";
+
+            var parser = new LegacyPipeParser();
+            var options = new ParserOptions { DefaultObx2Type = "ST" };
+
+            var parsed = (ORU_R01)parser.Parse(message, options);
+
+            var actualObservationValueInstance = parsed.GetPATIENT_RESULT(0).GetORDER_OBSERVATION(0).GetOBSERVATION(0).OBX.OBSERVATIONRESULTS.Data;
+
+            Assert.IsAssignableFrom(expectedObservationValueType, actualObservationValueInstance);
+            Assert.AreEqual("STValue", ((ST)actualObservationValueInstance).Value);
+        }
```

Comment:
Should we also check that the OBX-2 type in the parsed message has been set to `ST`?